.github/workflows/bindings_nodejs.yml

```diff
@@ -96,15 +86,23 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node }}
+          node-version: 18
           cache: yarn
           cache-dependency-path: "bindings/nodejs/yarn.lock"
       - name: Corepack
         run: corepack enable
       - name: Install dependencies
         run: yarn install
       - name: Build
-        run: yarn build
+        run: |
+          docker run \
+            -v ${{ github.workspace }}:/build \
+            -e NAPI_TARGET=x86_64-unknown-linux-gnu \
+            -w /build/bindings/nodejs \
+            ghcr.io/napi-rs/napi-rs/nodejs-rust:lts-debian \
+            bash -c "yarn build"
+          # change owner to current user
+          chown -R 1001:121 *.node
```

Comment:
What is the significance of doing so?

Reply:
> What is the significance of doing so?

files that generated by container is owned by `root`, we should change it back.

Reply:
Make sense.